src/views/v2/month/calendar-body/day.php

```diff
@@ -16,9 +16,9 @@
  * @var array $day The current day data.{
  *          @type string $date The day date, in the `Y-m-d` format.
  *          @type bool $is_start_of_week Whether the current day is the first day of the week or not.
- *          @type int $year_number The day year number, e.g. `2019`.
- *          @type int $month_number The day year number, e.g. `6` for June.
- *          @type int $day_number The day number in the month, e.g. `11` for June 11th.
+ *          @type string $year_number The day year number, e.g. `2019`.
```

Comment:
Why make these strings? I mean `$year_number`, `$month_number` and `$day_number`.

Reply:
Because we want to make sure we keep the leading `0`.